modules/microshift-storage-volume-cloning.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/creating-volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-storage-volume-cloning_{context}"]
+= About LVM volume cloning
+
+The LVMS supports `PersistentVolumeClaims` (PVC) cloning for LVM thin volumes. A clone is a duplicate of an existing volume at that can be used just like any other volume. When provisioned, an exact duplicate of the original volume is created when the `dataSource` references a source PVC in the same namespace. Once created, a cloned PVC is considered a new object and completely separate from the source PVC. The clone represents the data from the source at the moment in time it was created.
+
+[NOTE]
+====
+Cloning is only possible when the source and destination PVCs are in the same namespace. To create PVC clones, you must configure thin volumes on the {op-system-ostree} host.
+====
+
+== Use case example
+Volume cloning can be useful for creating the same data for multiple workloads. You can populate one volume with data downloaded over a network and then clone that volume instead of downloading the same dataset many times. For example, you can create a workload that downloads a data set. Next, you can create a replication controller with a PVC template that clones the source PVC. You can create a one-click action to scale the replicas up. The volume is then cloned automatically for each workload.
```

Comment:
@dhellmann made a good point in the gdoc that it's unlikely that users will be scaling workloads on MicroShift dynamically. And MicroShift isn't really intended fulfill a compute node role, so parallelizing data processing like in the example would be out of scope for the platform.

I realize I'm the one who provided this example in the first place 😅. Cloning is a feature of lvms, so we should definitely document it, but it's application on MicroShift is probably a lot more narrow than on OpenShift.

Reply:
@copejon What is a use case for a MicroShift user, then? Cloning data for multiple instances of MicroShift, perhaps?